### ISSUE TITLE
ng-Controller completion with available controllers

### ISSUE THREAD

**Issue by ganarajpr** [johnlindquist/angularjs-plugin#12]
The plugin should be able to provide code completion for ng-controller with options that can be picked from list of controllers.

**Comment by johnlindquist** [comment]
Agreed. Also looking into ways of navigating to that ctrl when you cmd+B (or cmd+click) on it.

**Comment by splurgy-doug** [comment]
:+1:  for cmd+click

**Comment by denofevil** [comment]
This one is fixed with JetBrains/intellij-plugins@0737139296c36bdf3b1c318ef0bf1a664ca8ea82
